crates/nargo_cli/tests/execution_success/array_eq/src/main.nr

```diff
@@ -0,0 +1,6 @@
+use dep::std;
+
+// Simple example of checking where two arrays are equal
+fn main(a: [Field; 32], b: [Field; 32]) {
+     std::assert_eq(a, b);
+}
```

Comment:
Note this reduces the amount of gates down to 33 compared to 128 with `assert(a == b)`